### ISSUE TITLE
Requirements too strict for ldap3 dependency

### ISSUE THREAD

**Issue by maaaaz** [dirkjanm/ldapdomaindump#19]
Hello @dirkjanm,

While installing the latest current `impacket` module, which [relies on `ldapdomaindump`](https://github.com/SecureAuthCorp/impacket/blob/master/requirements.txt), I'm getting that error message:
```
$ pip install impacket
...
ldapdomaindump 0.9.1 has requirement ldap3==2.5.1, but you'll have ldap3 2.5.2 which is incompatible.
...
```
I think that you can safely change your requirements from https://github.com/dirkjanm/ldapdomaindump/blob/cd5f862bab843f99a0bff1c38a2f5687ac366db0/setup.py#L9

to
```
      install_requires=['dnspython', 'ldap3>=2.5.1', 'future'],
```

Best regards.

**Comment by dirkjanm** [comment]
The latest impacket from git also has ldap3 forced on 2.5.1, reason for this is that 2.5.2 is currently broken, so this behavior is intentional. Once a new version of ldap3 is out this will be removed.

**Comment by maaaaz** [comment]
Ok, I didn't know this :)

**Comment by cannatag** [comment]
Hi, I’m the author of the ldap3 library. I want just to let you know that version 2.6 of ldap3 is available. Let me know if you still have problems with it.

Bye,
Giovanni

**Comment by dirkjanm** [comment]
Hi Giovanni, I did notice it but haven't had the time yet to run tests, thanks for the heads up, this is on my to-do.

**Comment by dirkjanm** [comment]
Unfortunately 2.6 still breaks for ldapdomaindump (but for a different reason which should occur less often for other tools), will have to stick with 2.5.1 until https://github.com/cannatag/ldap3/pull/675 is merged and released.
Will continue testing.

**Comment by exploide** [comment]
It appears the pull request has been merged and meanwhile ldap3 v2.6.1 has been released, which contains the fix.

Would be great to see the requirements relaxed. It currently causes package manager conflicts in distributions like Fedora.

**Comment by dirkjanm** [comment]
Thanks for the reminder. I just pushed ldapdomaindump 0.6.2 to pypi which has a looser requirement.
